### PR TITLE
♻️ add a separate descriptor pool for jtd

### PR DESCRIPTION
**Pull request by prashantgupta24**
Add a separate descriptor pool for jtd

### CODE REVIEW

**Comment by joerunde** [comment]
For context, this is what the protoc-generated _pb2 files do, and it seems to guard against duplicate descriptors

File: tests/test_jtd_to_proto.py [hunk 1]
```diff
@@ -208,20 +209,6 @@ def test_jtd_to_proto_enum(temp_dpool):
     assert fields["bat"].label == fields["bat"].LABEL_OPTIONAL
 
 
-def test_jtd_to_proto_top_level_enum(temp_dpool):
-    """Ensure that enums can be converted"""
```
Comment (prashantgupta24) [inline tests/test_jtd_to_proto.py:212]: this was a duplicated enum test

File: jtd_to_proto/jtd_to_proto.py [hunk 2]
```diff
@@ -16,6 +16,10 @@
 
 log = alog.use_channel("JTD2P")
 
+JTD_DESCRIPTOR_POOL = _descriptor_pool.DescriptorPool()
+# add google.protobuf.any to default descriptor pool for JTD
+JTD_DESCRIPTOR_POOL.AddSerializedFile(any_pb2.DESCRIPTOR.serialized_pb)
```
Comment (gabe-l-hart) [inline jtd_to_proto/jtd_to_proto.py:21]: I think we might also need timestamp here and possibly struct

**Review by gabe-l-hart — COMMENTED** [review]
One possible issue with adding timestamp and struct, but otherwise looks good

**Review by gabe-l-hart — APPROVED** [review]
Awesome, thanks Prashant!

**Pull request merged**